hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java

```diff
@@ -121,7 +121,7 @@ public void execute(OzoneClient client, OzoneAddress address)
       }
     }
 
-    if (replicationType != null) {
+    if(replicationType!=null) {
```

Comment:
Yes, I guess they got fixed as part of master merge to EC branch. Please rebase.

Reply:
I have reverted this change.